src/app/shared/example-viewer/example-viewer.ts

```diff
@@ -44,4 +50,12 @@ export class ExampleViewer {
   exampleFileUrl(extension: string) {
     return `/assets/examples/${this.example}-example-${extension.toLowerCase()}.html`;
   }
+
+  copySource(text: string) {
+    if (this.copier.copyText(text)) {
+      this.snackbar.open('Code copied', '', {duration: 1000});
```

Comment:
I think that `2500ms` is probably a better duration, since `1000ms` essentially is enough time to appear and then immediately hide.